src/lib/store.js

```diff
@@ -49,7 +49,8 @@ const initialState = {
   // cookie policy.
   // We automatically accept cookies in dev and test environments so the cookie
   // banner doesn't interfere with tests.
-  cookiePreference: isProd ? null : 'accepts',
+  // cookiePreference: isProd ? null : 'accepts',
+  cookiePreference: null,
```

Comment:
Reverse this change before merging:

```suggestion
  cookiePreference: isProd ? null : 'accepts',
```